samples/21_sound_and_effects.cr

```diff
@@ -22,35 +26,38 @@ activekey = [] of LibSDL::Keycode
 loop do
   case event = SDL::Event.wait
   when SDL::Event::Quit
-    #SDL::Mixer.unload_music(music)
+    music.stop
+    SDL::MIX.close
```

Comment:
Do we have to close the mixer? The `at_exit` above will already call `MIX.quit`.